contracts/Cauldron.sol

```diff
@@ -3,16 +3,10 @@ pragma solidity >=0.8.13;
 import "./interfaces/IFYToken.sol";
 import "./interfaces/IOracle.sol";
 import "./interfaces/DataTypes.sol";
-import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
-import "@yield-protocol/utils-v2/contracts/math/WMul.sol";
-import "@yield-protocol/utils-v2/contracts/math/WDiv.sol";
-import "@yield-protocol/utils-v2/contracts/math/WDivUp.sol";
-import "@yield-protocol/utils-v2/contracts/cast/CastU128I128.sol";
-import "@yield-protocol/utils-v2/contracts/cast/CastI128U128.sol";
-import "@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol";
-import "@yield-protocol/utils-v2/contracts/cast/CastU256U32.sol";
-import "@yield-protocol/utils-v2/contracts/cast/CastU256I256.sol";
 import "./constants/Constants.sol";
+import "../lib/yield-utils-v2/src/access/AccessControl.sol";
+import "../lib/yield-utils-v2/src/utils/Math.sol";
+import "../lib/yield-utils-v2/src/utils/Cast.sol";
```

Comment:
why no remappings?

Reply:
Because I want to use submodules in the `environments-v2` repo